javascripts/content_scripts/privly.js

```diff
@@ -763,8 +754,6 @@ var privly = {
 
     "use strict";
 
-    return;
```

Comment:
This probably should have been commented, but I believe it needs to remain here.